src/hooks/useProductList.js

```diff
@@ -0,0 +1,60 @@
+const useProductList = () => {
+  const fakeProductsList = [
+    {
+      id: '1',
+      productPhotoUrl: 'https://placehold.co/310x210?text=Imagen',
+      listPrice: '250.000,00',
+      discountedPrice: '200.000,00',
+      discountPercentage: 20,
```

Comment:
Roque mentioned me something about this today (formatting)--> Data should be formatted here (the Presenter) so that the Views only render their props. Since this is fake data, you can put the formatting directly.

So, we should remove the formatting bits from the Views, for example `$` and some `%`. and put them here.